src/tests/java/picard/illumina/parser/CycleIlluminaFileMapTest.java

```diff
@@ -75,42 +75,4 @@ private static String constructPathString(int lane, int cycle) {
             },
         };
     }
-
-  /*  @Test(dataProvider = "iteratorTestData")
```

Comment:
@yfarjoun Do you know why these tests are commented out?  Should they be uncommented and disabled instead of removing?